src/BlurView.android.js

```diff
@@ -1,51 +1,73 @@
 import React, { Component, PropTypes } from 'react';
-import { View, requireNativeComponent, DeviceEventEmitter, } from 'react-native';
-
-const OVERLAY_COLORS = {
-  light: 'rgba(255, 255, 255, 0.2)',
-  xlight: 'rgba(255, 255, 255, 0.75)',
-  dark: 'rgba(16, 12, 12, 0.64)',
-};
+import {
+  View,
+  requireNativeComponent,
+  DeviceEventEmitter,
+  findNodeHandle,
+} from 'react-native';
 
+import { overlayColorForProps, blurRadiusForProps } from './lib/ios_blur_view_emulation';
```

Comment:
Can `ios_blur_view_emulation` follow our naming convention? We use lowerCamelCase for that kind of files. 

Also, I'm not sure if this name reflects the real purpose of the file: in the title you say "ios...", but in fact, I think it's needed for other platforms.

Probably, it'll be better if we can use something more neutral? `blurViewHelpers`?